### DISCUSSION TITLE
missing from Command Search : File > most recent maps

### DISCUSSION THREAD

**Discussion by kenfhill84083** [freeplane/freeplane#2146] [Help]
I invoked Command Search and typed "most recent maps" + different combinations of the individual words - but nothing came up to help me find File > Most recent maps.  I like to keep the menu bar hidden and use Command Search exclusively.

Is this as designed?  any suggestions?

**Comment by dpolivaev** [comment]
Currently command search does not include variable content like recent maps or node styles.

**Comment by kenfhill84083** [comment]
Thanks for the clarification.  Would it be possible to include the point at which the menus jump into those variable content things -- but have Command Search display them as greyed out or somesuch? 

I use CS partly to discover where menu items are.  Most recent maps is an easy one to remember but I have been away from Freeplane for a while and since it's introduction CS has become my primary way of finding and doing things in FP.  I tend to hide the menus and depend on CS for non-hotkey functionality.

Freeplane is great but this would it even better, for me at least.  But I know you are busy and this is minor.
